ShoppIt/src/resources/fxml/IndividualFruitList.fxml

```diff
@@ -6,22 +6,36 @@
 <?import javafx.scene.layout.ColumnConstraints?>
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.text.Font?>
 
 <AnchorPane onMouseClicked="#click" prefHeight="50.0" prefWidth="760.0" stylesheets="@../css/application.css" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="controllers.IndividualFruitListController">
    <children>
       <GridPane fx:id="cardGridPane" prefHeight="50.0" prefWidth="591.0">
         <columnConstraints>
-          <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
+          <ColumnConstraints hgrow="SOMETIMES" maxWidth="492.0" minWidth="10.0" prefWidth="382.8" />
+            <ColumnConstraints hgrow="SOMETIMES" maxWidth="492.0" minWidth="10.0" prefWidth="141.40000000000003" />
+            <ColumnConstraints hgrow="SOMETIMES" maxWidth="284.79998779296875" minWidth="10.0" prefWidth="67.99997558593759" />
         </columnConstraints>
         <rowConstraints>
           <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
         </rowConstraints>
          <children>
-            <Label fx:id="fruitNameLabel" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" prefHeight="30.0" prefWidth="710.0" text="Fruit Name" textFill="#eeeeee">
+            <Label fx:id="fruitNameLabel" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" prefHeight="30.0" prefWidth="710.0" text="Fruit Name" textFill="#eeeeee" GridPane.columnIndex="0">
                <GridPane.margin>
                   <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
                </GridPane.margin>
+               <font>
+                  <Font name="Dubai Bold" size="12.0" />
+               </font>
             </Label>
+            <Label fx:id="priceLabel" alignment="CENTER" contentDisplay="CENTER" prefHeight="66.0" prefWidth="162.0" text="Price" textFill="#eeeeee" GridPane.columnIndex="2">
+               <font>
+                  <Font name="Dubai Bold" size="12.0" />
+               </font></Label>
+            <Label fx:id="calorieLabel" alignment="CENTER" prefHeight="50.0" prefWidth="152.0" text="Calories" textFill="#eeeeee" GridPane.columnIndex="1">
+               <font>
+                  <Font name="Dubai Bold" size="12.0" />
+               </font></Label>
```

Comment:
I think some properties here could be added to the css stylesheet (rather than individual fxml elements) so that they can be reused if the application were to continue to expand. For example, the size, font and colour of the text. Perhaps give this a go?